### ISSUE TITLE
Download in higher Resolution

### ISSUE THREAD

**Issue by Bennibaba** [luckyrat/KeePass-Favicon-Downloader#22]
Hello,
thank you for the great Plugin.
Would it be possibly to include an option to download the Favicons in a higher resolution.
At the current state, the pictures looks very bad on a phone screen.

Thank you very much!

**Comment by luckyrat** [comment]
Yeah probably. Assuming KeePass supports higher resolution icons now?

When I wrote this plugin there was only one resolution of icon on the internet but I wouldn't be surprised if we can increase the resolution now.

Do you have any examples of icons that look better on their website than in the app you're using for your KeePass database?

What app do you use on your phone? Does it support higher resolution icons?

**Comment by tomillie** [comment]
I use Keepass2Android and all icons downloaded using the plugin have poor resolution. The app supports higher resolutions, because those I downloaded manually look good in the app.

**Comment by joshivince** [comment]
I use KeePass Touch for iOS and yes, some of the icons are a little 'blurry'. But I can live with it... just to say...
